awx/ui_next/src/screens/ObservabilityMetrics/ObservabilityMetrics.jsx

```diff
@@ -0,0 +1,239 @@
+  const {
+    result: { instances, metrics },
+    error: fetchInitialError,
+    request: fetchInstances,
+  } = useRequest(
+    useCallback(async () => {
+      const [
+        {
+          data: { results },
+        },
+        { data: mets },
+      ] = await Promise.all([
+        InstancesAPI.read(),
+        MetricsAPI.read({
+          subsystemonly: 1,
+          format: 'json',
+        }),
+      ]);
+
+      const metricOptions = Object.keys(mets);
+
+      return {
+        instances: [...results.map(result => result.hostname), 'All'],
+        metrics: metricOptions,
+      };
+    }, []),
+    { instances: [], metrics: [] }
+  );
+
+  const {
+    result: helpText,
+    error: updateError,
+    request: fetchData,
+  } = useRequest(
+    useCallback(async () => {
+      const { data } = await MetricsAPI.read({
+        subsystemonly: 1,
+        format: 'json',
+        node: instance === 'All' ? null : instance,
+        metric,
+      });
+
+      const rendered = renderedData;
+      const instanceData = Object.values(data);
+      instanceData.forEach(value => {
+        value.samples.forEach(sample => {
+          instances.forEach(i => {
+            if (i === sample.labels.node) {
+              const renderedIndex = renderedData.findIndex(rd => rd.name === i);
+
+              if (renderedIndex === -1) {
+                rendered.push({
+                  name: i,
+                  values: [
+                    {
+                      y: sample.value,
+                      x: count.length - 1,
+                    },
+                  ],
+                });
+              } else if (
+                rendered[renderedIndex].values?.length === 0 ||
+                !rendered[renderedIndex].values
+              ) {
+                rendered[renderedIndex].values = [
+                  { y: sample.value, x: count.length - 1 },
+                ];
+              } else {
+                rendered[renderedIndex].values = [
+                  ...rendered[renderedIndex].values,
+                  { y: sample.value, x: count.length - 1 },
+                ];
+              }
+            }
+          });
+        });
+      });
+      let countRestrictedData = rendered;
+      if (count.length > 49) {
+        countRestrictedData = rendered.map(({ values, name }) => ({
+          name,
+          values: values.slice(-50),
+        }));
+      }
+
+      setRenderedData(countRestrictedData);
+      return data[metric].help_text;
+
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [instance, metric, instances]),
+    ''
+  );
+
+  useInterval(fetchData, 3000, instance, metric);
+
+  useEffect(() => {
+    if (instance && metric) {
+      fetchData();
+    }
+  }, [fetchData, instance, metric]);
+
+  useEffect(() => {
+    fetchInstances();
+  }, [fetchInstances]);
+  if (fetchInitialError || updateError) {
+    console.log('here');
+    return (
```

Comment:
Breadcrumbs are missing:

![Screenshot from 2021-04-06 09-52-37](https://user-images.githubusercontent.com/9753817/113723682-9523bf80-96bf-11eb-87a5-317f8781e30c.png)

Reply:
Hey. @jakemcdermott , @AlexSCorey - just wanted to see if there's an update on this and the other requested change.